app/components/c-linklist.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <ul :class="b('list')">
```

Comment:
- [ ] No need for a block, just use b('')

Reply:
I need a css class for the list.